src/lib/dashboard-model.js

```diff
@@ -3,17 +3,31 @@
 const config = require('../config');
 const query = config.postgres.mock ? require('../../mock/mock-query') : require('./db/query');
 
-const costSaving = () => query.totalCustomerSearches()
-  .then(searches => Math.round(((searches / 2) + 793235) * 1.81));
+/**
+ * Martin's Multiplier `M`, an estimated ratio of relevant searches in the audit table, currently 55%
+ * @type {number}
+ */
+const M = 0.55;
+/**
+ * Cost of a Second Class "Signed for" letter from Royal Mail: £1.81
+ * @type {number}
+ */
+const p = 1.81;
+const Mp = M * p;
+/**
+ * costSaving calculates the approximate cost saving to the public from not having to send certificates by post
+ * @param T Total searches done to date
+ * @returns {number} the approximate cost saving to the public
+ */
+const costSaving = T => Math.round(T * Mp);
```

Comment:
I'm a little concerned about introducing JSDoc when we don't use it anywhere else in the code-base.

Reply:
I think I would prefer meaningful variable names with some very minimal comment when required.

e.g.
```js
const fractionOfQueries = 0.55; // Estimate of fraction of queries that correspond to a single, real search.
const costOfPostage = 1.81; // Second Class "Signed for" letter from Royal Mail: £1.81
const queriesToSaving = T => Math.round(T * fractionOfQueries * costOfPostage);
```
```

Reply:
See latest commit